assets/src/theme/useServerThemes.js

```diff
@@ -24,13 +24,17 @@ const deleteTheme = async (name) => {
 export const useServerThemes = () => {
   const [serverThemes, setServerThemes] = useState([]);
   const [dirty, setDirty] = useState(false);
+
   useEffect(() => {
     const doApiCall = async () => {
       const themes = await wp.apiFetch({
         path: 'planet4/v1/themes/',
         method: 'GET',
       });
-      setServerThemes(themes);
+      setServerThemes({
+        'default': {},
```

Comment:
I added this as otherwise there's no easy way to start a new theme from the default theme.